### DISCUSSION TITLE
trivy maven repo access behind firewall

### DISCUSSION THREAD

**Discussion by pepe-pj** [aquasecurity/trivy#5914] [Q&A]
### Question

Hi,

my trivy repo scan run behind a firewall, and trivy can't go to [repo.maven.apache.org](http://repo.maven.apache.org/) directly.
This significantly slows down the running of trivy repo scan behind the firewalls, and cannot retrive the library's metadata (pom.xml etc) and the the library vulnerabilty check is limited.

can I configure it somehow so that I don't connect directly to official maven, but i want to connect to maven official repo through the local maven proxy (nexus)?

Thanx,
Peter

### Target

Git Repository

### Scanner

Vulnerability

### Output Format

Table

### Mode

Standalone

### Operating System

RHEL 7, docker

### Version

```bash
Version: 0.48.1
```

**Comment by GulShow** [comment]
Hi, y'all! Could you please let me know if you found a solution?

**Comment by DmitriyLewen** [comment]
Hello @pepe-pj @GulShow 
Sorry for delay.

Take a look #8118.

Regards, Dmitriy
